### PR TITLE
Remove localhost references in docs

### PR DESCRIPTION
**Pull request by wojtczakmat**
I changed URL from localhost to correct GitHub docs

### CODE REVIEW

File: docs/src/examples/form/index.mdx [hunk 1]
```diff
@@ -75,7 +75,7 @@ abstract class _FormStore with Store {
 
 Validations on the fields is not an explicit action that is invoked on the `FormStore`. Instead, it is considered a side-effect of a change in one of the fields. In MobX, we can model such side-effects with **`reactions`**.
 
-Reactions come in a few flavors but we are going to focus on the aptly named [`reaction(selector, effect)`](http://localhost:3000/api/reaction#reaction) that takes in two arguments. The first one is a _selector-function_ that monitors the observables used within it. It is supposed to return a value that is compared with a previously returned value. When it changes, it will invoke the _effect-function_ (the second argument).
+Reactions come in a few flavors but we are going to focus on the aptly named [`reaction(selector, effect)`](https://github.com/mobxjs/mobx.dart/blob/master/docs/src/api/reaction.mdx#reaction) that takes in two arguments. The first one is a _selector-function_ that monitors the observables used within it. It is supposed to return a value that is compared with a previously returned value. When it changes, it will invoke the _effect-function_ (the second argument).
```
Comment (pavanpodila) [inline docs/src/examples/form/index.mdx]: Thanks for catching this! This url is for the website. It should really be `/api/reaction#reaction`

Comment (wojtczakmat) [reply docs/src/examples/form/index.mdx]: Got it, now it should be all right

**Comment by wojtczakmat** [comment]
OK, maybe let's expand this PR to removing all localhost appearances

**Comment by wojtczakmat** [comment]
Now there's no `localhost` search results in the repo

**Pull request merged**